tox.ini

```diff
@@ -1,19 +1,30 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-lint
+    py{27,35}-ansible22-unit
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    wheel
```

Comment:
I'm reworking it to push the version dependent installs to the `commands` section to remove the conflict.